service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -32,6 +32,8 @@ oc project istio-system
 $ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
 ----
 +
+. Check your {ProductName} 1.1 `ServiceMeshControlPlane` resource configuration to make sure it's valid. If it's invalid, or parts that are invaild, you must fix any invalid configuration items before continuing. See xref:../../service_mesh/v2x/customizing-installation-ossm.html#ossm-updating-smcp_customizing-installation-ossm-v2x[Updating the `ServiceMeshControlPlane`].
```

Comment:
I don't think there's enough detail here.  To fix the issue, users need to specifically edit/modify the v1 version of the resource, which means they must either update the resource (could be v1 or v2), patch the resource (must be a v1 patch), or edit the resource (must be v1).  Some examples:

```bash
$ oc get smcp.v1.maistra.io bad-resource > smcp-resource.yaml
#... edit and fix smcp-resource.yaml
$ oc replace -f smcp-resource.yaml

$ oc patch smcp.v1.maistra.io bad-resource --type json --patch '[{"op": "replace","path":"/spec/path/to/bad/setting","value":"good-value"}]'

$ oc edit smcp.v1.maistra.io bad-resource
```